### ISSUE TITLE
some questions on the actor_critic.py file

### ISSUE THREAD

**Issue by wanyao1992** [ShangtongZhang/DeepRL#2]
Hi Shangtong,

Sorry to trouble again. Recently I am reading your source code, especially on the A3C part. I am a bit confused on the following part.

```
GAE = torch.FloatTensor([[0]])
for i in reversed(range(len(pending))):
    prob, log_prob, value, action, reward = pending[i]
    if i == len(pending) - 1:
        delta = reward + config.discount * R - value.data
    else:
        delta = reward + pending[i + 1][2].data - value.data
    
    GAE = config.discount * config.gae_tau * GAE + delta
    loss += -log_prob.gather(1, Variable(torch.LongTensor([[action]]))) * Variable(GAE)
    loss += config.entropy_weight * torch.sum(torch.mul(prob, log_prob))
    R = reward + config.discount * R
    loss += 0.5 * (Variable(R) - value).pow(2)
```

1. Which paper do you refer on implementing the A3C, since I want to know the detailed equation of calculating GAE. I don't know the gae_tau mean, since in the paper I have read, there is no such symbol, so I am a bit confused.

2. I can't understand the way of calculating loss. Could you give some comments on this part. 

Since I am a beginner in reinforcement learning, maybe I will encounter more problem when I finish reading your code. It's highly appreciated if you can help me or give me some suggestions. Thanks.

**Comment by ShangtongZhang** [comment]
1. If you read all the referenced papers in the list in README, you can find `GAE` is in one of them, even if you only read the title of those papers.
2. No. I don't have time to comment my code unless bugs.
